e2e-tests/development-runtime/gatsby-node.js

```diff
@@ -1,6 +1,75 @@
 const path = require(`path`)
 const { createFilePath } = require(`gatsby-source-filesystem`)
+const {
+  addRemoteFilePolyfillInterface,
+  polyfillImageServiceDevRoutes,
+} = require("gatsby-plugin-utils/polyfill-remote-file")
 
+/** @type{import('gatsby').createSchemaCustomization} */
+exports.createSchemaCustomization = ({ actions, schema, store }) => {
+  actions.createTypes(
+    addRemoteFilePolyfillInterface(
+      schema.buildObjectType({
+        name: "MyRemoteFile",
+        fields: {},
+        interfaces: ["Node", "RemoteFile"],
+      }),
+      {
+        store,
+        schema,
+      }
+    )
+  )
+}
+
+/** @type {imporg('gatsby').sourceNodes} */
+exports.sourceNodes = ({ actions, createNodeId, createContentDigest }) => {
+  const items = [
+    {
+      name: "photoA.jpg",
+      url: "https://images.unsplash.com/photo-1517849845537-4d257902454a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2000&q=80",
```

Comment:
Are those "stable" URLs that won't break our tests in the future?

Reply:
They should be stable yes